manifests/params.pp

```diff
@@ -77,9 +77,8 @@
 
   # The aio puppet-agent version currently installed on the compiling master
   # (only used in PE)
-  $master_agent_version = $_is_pe ? {
-    true    => pe_compiling_server_aio_build(),
-    default => undef,
+  if ($_is_pe && is_function_available('pe_compiling_server_aio_build')) {
+    $master_agent_version = pe_compiling_server_aio_build()
```

Comment:
Can we add an else to explicitly set it to `undef`? We're going to get warnings all over the place otherwise.